marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/AbstractSparkConnectorTest.java

```diff
@@ -52,10 +52,26 @@ protected File getTestProjectDirectory() {
     @Override
     protected HubConfigImpl runAsUser(String username, String password) {
         hubProperties = new Properties();
-        // Assumes localhost for now; this will be improved once this project is developed on the 5.4 branch
-        hubProperties.setProperty("mlHost", "localhost");
+        String mlHost="localhost";
+        String hubDHS="false";
```

Comment:
For consistency with other DHF props, do "hubDhs" and "isDhs" (that's consistent with "Ssl", for example).

Reply:
Addressed in new the commit.